Spatial_Engine/Spatial_Engine.csproj

```diff
@@ -32,27 +32,30 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="BHoM">
-<HintPath>C:\ProgramData\BHoM\Assemblies\BHoM.dll</HintPath>
+      <HintPath>C:\ProgramData\BHoM\Assemblies\BHoM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Data_oM">
+      <HintPath>..\..\..\ProgramData\BHoM\Assemblies\Data_oM.dll</HintPath>
```

Comment:
the path should not be relative?

Reply:
Yes, it's a new world, but should already been fixed